features/catalog/step_definitions/document-upload.ts

```diff
@@ -43,6 +49,7 @@ When(
       descriptorId: z.string(),
     });
 
+    // to do: this shouldn't use dataPreparationService
```

Comment:
cc @AsterITA

Reply:
Good catch, I agree, can you include it in this PR?